java/README.md

```diff
@@ -9,10 +9,11 @@ to develop this Java wrapper.
 
 The Java client contains the following parts:
 
-1. `client`: A Java-wrapper around the rust-core client.
-2. `examples`: An examples app to test the client against a Redis localhost
-3. `benchmark`: A dedicated benchmarking tool designed to evaluate and compare the performance of GLIDE for Redis and other Java clients.
-4. `integTest`: An integration test sub-project for API and E2E testing
+1. `src`: A rust part of the Java client and corresponding tests.
```

Comment:
```suggestion
1. `src`: Rust dynamic library FFI to integrate with [GLIDE core library](../glide-core/README.md). 
```
Not sure this is needed. 
This 'part' goes hand-in-hand with the client.  We should consider moving it into the client directory, actually.